### ISSUE TITLE
Add new "Events" methods

### ISSUE THREAD

**Issue by aglitchman** [indiesoftby/defold-yagames#11]
**Documentation:** https://yandex.ru/dev/games/doc/dg/sdk/sdk-events.html

**Comment by aglitchman** [comment]
API could be:

- `yagames.event_dispatch(name)`
- `yagames.event_on(name, listener)` to set global listener for the event with `name`.

JavaScript glue code could handle event `name` like that:
```js
// current supported events: "HISTORY_BACK", "EXIT"
ysdk.EVENTS[name]
```

**Comment by aglitchman** [comment]
Implemented in 9a4259852bce9d3d95f219d5f8adca1e8890bc3f
